plat/nvidia/tegra/common/drivers/bpmp/bpmp.c

```diff
@@ -0,0 +1,197 @@
+/*
+ * Copyright (c) 2017, ARM Limited and Contributors. All rights reserved.
```

Comment:
Use SPDX identifier. Also, for new files you can simply put Nvidia as copyright holder.